site/docs/reference/Connectors/capture-connectors/HubSpot.md

```diff
@@ -1,9 +1,9 @@
 
-# Hubspot
+# HubSpot
```

Comment:
This probably should match their actual name as well given they are not approving a URL.